src/olympia/files/models.py

```diff
@@ -63,6 +63,9 @@ class File(OnChangeMixin, ModelBase):
     status = models.PositiveSmallIntegerField(
         choices=STATUS_CHOICES.items(), default=amo.STATUS_AWAITING_REVIEW)
     datestatuschanged = models.DateTimeField(null=True, auto_now_add=True)
+    is_restart_required = models.BooleanField(default=False)
+    # FIXME: remove no_restart once is_restart_required is deployed to
+    # production.
```

Comment:
I created https://github.com/mozilla/addons-server/issues/5881 ; I removed the field entirely in order to prepare for the removal of the column.